examples/mul.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
+     return 1 + 2 * 3
```

Comment:
After https://github.com/eatonphil/goi/commit/5716ad33c3945da04bee571e5bd5bb03e9884353 this should be a println instead of a return.

Reply:
I've rebased and updated to use `println` now, thanks :)